src/Compilers/CSharp/Test/Syntax/Parsing/ParserErrorMessageTests.cs

```diff
@@ -3348,7 +3328,7 @@ public class clz
 }
 ";
 
-            ParseAndValidate(test, Diagnostic(ErrorCode.ERR_MemberNeedsType, "x"));
+            ParseAndValidate(test);
```

Comment:
We have semantic tests in the semantic test assembly that checks for those diagnostics. This is a syntax test.